requirements.txt

```diff
@@ -0,0 +1,6 @@
+-e git+https://github.com/nickumia-reisys/ckanext-harvest.git@9d1f647d247c16b6c3acba26e321e9500cafb18c#egg=ckanext-harvest
```

Comment:
Can you describe why we have to be on a personal git commit? Should we merge this to GSA fork?